backend/routes/Department.go

```diff
@@ -0,0 +1,25 @@
+package routes
+
+import (
+	"net/http"
+
+	"github.com/ucsdscheduleplanner/UCSD-Schedule-Planner/backend/db"
+)
+
+const logTagDepartment = "[Department]"
+
+// GetDepartments is a pre-http.HandlerFunc for department route and will become a closure with *DatabaseStruct
+func GetDepartments(writer http.ResponseWriter, request *http.Request, ds *db.DatabaseStruct) {
+
+	if request.Method != "GET" {
+		errInvalidMethod(logTagDepartment, writer, request)
+		return
+	}
+
+	queryAndResponse(
```

Comment:
these are a lot of arguments and I"m not quite sure what they all do, you should try and refactor this